test/test/mocks/widgets.js

```diff
@@ -98,21 +51,22 @@ export const WIDGET_LIST = {
     ],
   },
 };
+
 export const WIDGETS_MAP = WIDGET_LIST.payload.reduce((acc, widget) => {
   acc[widget.code] = widget;
   return acc;
 }, {});
+
 export const WIDGET_ONE_LIST = {
-  'User Widget': [
-    {
-      widgetCategory: 'User Widget',
-      code: 'WTF',
-      name: 'My first  Widget',
-      used: 0,
-      titles: {
-        it: 'Mio Widget',
-        en: 'My Widget',
-      },
-    },
-  ],
+  'User Widget': [WIDGET],
+};
+
+export const WIDGET_POST_PUT = {
```

Comment:
What is the meaning of this POST_PUT suffix? Can't this mock be replaced by WIDGET in the files that are using it?